### ISSUE TITLE
[bug]: Mail not send when adding members

### ISSUE THREAD

**Issue by Sarank015** [makeplane/plane#3311]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When I add member to the workspace, In the UI, it says mail has been sent successfully but it's not send to the corresponding mail ID. 

### Steps to reproduce

1. Go to Workspace Setting and go to Members tab
2. Add a new member with their email ID. It says the mail has been sent successfully, but the mail is didn't received to that corresponding mail ID.
3. What are the configurations need to be done to rectify the issue

### Browser

Google Chrome

### Version

Cloud

**Comment by pablohashescobar** [comment]
@Sarank015, we checked this issue in [cloud](https://app.plane.so) and we were unable to replicate it. If you are using the self hosted version you can configure your smtp settings in the `god-mode`

**Comment by Sarank015** [comment]
Hi @pablohashescobar 

I have installed your open-source service "Plane" for our Param organization. Initially, it was working with an issue of sending invitation e-mails while inviting new users. In the process of fixing this issue, we upgraded/re-installed your latest version. We followed the exact steps as per your documentation and all the steps were successful. However, the page does not load and is struck as below.

Can I request you for a call to help us resolve this issue?

**Comment by pablohashescobar** [comment]
Hey @Sarank015, can you jump on to our discord we can talk there.

**Comment by mark0wka** [comment]
I have a similar issue with sending emails.

**Comment by BentHaase** [comment]
For some reason I had to use port 587 for Plane, even though every other client works just fine with 465 and my mailserver. When using port 465 I saw a connection request in my mailserver logs but nothing more than that happened, no mails were sent even though plane pretended 'everything is fine'.

Is there any way to see more in-depth worker logs?

This is what I got using `docker compose logs -f` when Plane was pretending everything is fine, no error and no mail was sent:

```bash
plane<someproject>exampletld-api-1          | 172.23.0.10:46666 - "GET /api/workspaces/<someproject>/projects/ HTTP/1.0" 200
plane<someproject>exampletld-proxy-1        | 172.23.0.1 - - [28/Jan/2024:19:23:33 +0000] "GET /api/workspaces/<someproject>/projects/ HTTP/1.1" 200 799 "https://plane.<someproject>.example.tld/<someproject>/settings/members" "Mozilla/5.0 (X11; Linux x86_64; rv:122.0) Gecko/20100101 Firefox/122.0"
plane<someproject>exampletld-proxy-1        | 172.23.0.1 - - [28/Jan/2024:19:23:33 +0000] "POST /<someproject>/settings/1/decide/?v=3&ip=1&_=1706469813306&ver=1.93.6 HTTP/1.1" 308 74 "https://plane.<someproject>.example.tld/<someproject>/settings/members" "Mozilla/5.0 (X11; Linux x86_64; rv:122.0) Gecko/20100101 Firefox/122.0"
plane<someproject>exampletld-proxy-1        | 172.23.0.1 - - [28/Jan/2024:19:23:33 +0000] "POST /<someproject>/settings/1/decide?v=3&ip=1&_=1706469813306&ver=1.93.6 HTTP/1.1" 404 2022 "https://plane.<someproject>.example.tld/<someproject>/settings/members" "Mozilla/5.0 (X11; Linux x86_64; rv:122.0) Gecko/20100101 Firefox/122.0"
plane<someproject>exampletld-worker-1       | [2024-01-28 19:23:33,372: INFO/MainProcess] Task plane.bgtasks.workspace_invitation_task.workspace_invitation[yyyyyyyy-xxxx-xxxx-xxxx-xxxxxxx] received
plane<someproject>exampletld-api-1          | 172.23.0.10:46698 - "POST /api/workspaces/<someproject>/invitations/ HTTP/1.0" 200
plane<someproject>exampletld-proxy-1        | 172.23.0.1 - - [28/Jan/2024:19:23:33 +0000] "POST /api/workspaces/<someproject>/invitations/ HTTP/1.1" 200 38 "https://plane.<someproject>.example.tld/<someproject>/settings/members" "Mozilla/5.0 (X11; Linux x86_64; rv:122.0) Gecko/20100101 Firefox/122.0"
```

**Comment by iqbalabd** [comment]
We've set up a self-hosted instance of Plane (v0.14.0) and I have set my email server in .env and it through /god-mode/.

We're using `smtp.sendgrid.net` and have set the ports, user and password credentials. These work fine for our other stuff, but it seems the user invitation emails are not even reaching Sendgrid from the Plane instance. We don't know the issue, so can we look into logs somewhere and try debugging things?

**Comment by mmospanenko** [comment]
yes, having the same issue, no any debug information, Celery tasks executes successfully. Will be great to add Test button for settings form

**Comment by dsmh** [comment]
> We've set up a self-hosted instance of Plane (v0.14.0) and I have set my email server in .env and it through /god-mode/.
> 
> We're using `smtp.sendgrid.net` and have set the ports, user and password credentials. These work fine for our other stuff, but it seems the user invitation emails are not even reaching Sendgrid from the Plane instance. We don't know the issue, so can we look into logs somewhere and try debugging things?

I'm having the very same problem here.

**Comment by pablohashescobar** [comment]
Hi, 

Sorry for the late reply we are working on adding a test email button in the god mode for email settings which will display the error in the UI itself. In addition to that we have added another [utility script](https://github.com/makeplane/plane/blob/develop/apiserver/plane/db/management/commands/test_email.py) to check the email connection from the shell directly.

**Comment by dsmh** [comment]
> Hi,
> 
> Sorry for the late reply we are working on adding a test email button in the god mode for email settings which will display the error in the UI itself. In addition to that we have added another [utility script](https://github.com/makeplane/plane/blob/develop/apiserver/plane/db/management/commands/test_email.py) to check the email connection from the shell directly.

Thank you much. May you please put an examble on how to run this script from the shell?

**Comment by pablohashescobar** [comment]
I am sharing the snippet here. We will update this in the docs as well.
```
python manage.py test_email "<email>"
```

**Comment by ZerNico** [comment]
@pablohashescobar I actually have the same issue but always got `timed out`.

It looks like the `Turn SSL on` toggle does not get saved. When toggling it on in god mode and refreshing it is back to disabled. Also I tried editing the test_mail script to manually include ssl, then it works.

**Comment by mmospanenko** [comment]
Issue was with IP, we should pass server IP as env argument if server is behind CloudFlare/Traefik.

EMAIL_TRANSPORT_DEFAULT_CLIENT

Also I had the issue with keys, I generated from Pro version, but used Ce, mail service warned about this.

And very strange that Test message was ok from the ui, it has to use the same implementation.

**Comment by Lermatroid** [comment]
Hey @mmospanenko, what did you wind up having to do with cloudflare? We are having a issue where emails wont send but have our instance behind CF.

Thanks!

**Comment by pablohashescobar** [comment]
@Lermatroid, can you check by running the included test script `python manage.py test_email <receiver_email>`

**Comment by mmospanenko** [comment]
Sorry for confusion, I think I was missed with the thread. Doesn't work for me(

**Comment by kordeviant** [comment]
in onpremise godmode smtp test email is sent and working , but inviting memebers to workspace shows pending but nothing is sent totheir email

**Comment by PabloGmz96** [comment]
> I am sharing the snippet here. We will update this in the docs as well.
> 
> ```
> python manage.py test_email "<email>"
> ```

Just for reference, in order to execute this command to test your already-saved SMTP settings you should enter to Host server of your Plane instance, go to path where `docker-compose.yml` is and execute the following command:
```sh
docker compose exec api bash
```
This means that docker will enter you to an interactiva bash shell running into the API image service. There you'll be able to run the former command.
Hope it helps!

**Comment by pushya22** [comment]
@Sarank015 Hope the issue is resolved, feel free to re-open in case if you are still facing it.
